doc/source/cluster/running-applications/job-submission/cli.rst

```diff
@@ -16,6 +16,12 @@ This section contains commands for :ref:`Ray Job Submission <jobs-quickstart>`.
     ``ray job submit --working_dir="." -- python script.py`` instead of ``ray job submit --working_dir="." -- "python script.py"``.
     Otherwise you may encounter the error ``/bin/sh: 1: python script.py: not found``.
 
+.. warning::
+
+   The entrypoint command must be provided last, and any arguments to `ray job submit` must be provided before the entrypoint command.
```

Comment:
```suggestion
   You must provide any arguments to `ray job submit` before the entrypoint command and the entrypoint command must be last.
```

Reply:
This warning seems to contradict the example in the previous warning about not wrapping the entrypoint command in quotes. I may be confused, but I wanted to point it out, just in case.

Reply:
Oh, I guess it depends whether we consider the entrypoint command to be an argument to `ray job submit`.  

How about this?


> You must provide the entrypoint command last (after the ` -- `), and any other arguments to `ray job submit` (e.g. `--working_dir="."`) must be provided before the ` -- `.

Reply:
I like that! Just put a comma after `e.g.,`

I was confused because I didn't realize the entrypoint command was `python script.py`.  If it's not too rudimentary, I would suggest mentioning that, something like this:
```You must provide the entrypoint command, `python script.py`, last (after the --), and any other arguments to `ray job submit` (that is, --working_dir=".") must be provided before the  two hyphens (--).
```

Reply:
Makes sense! Updated, @angelinalg feel free to approve the PR if it looks okay.